src/usr/lib/rsetup/cli/system.sh

```diff
@@ -17,6 +17,21 @@ ALLOWED_RCONFIG_FUNC+=(
     "set_led_netdev"
 )
 
+get_setup_script() {
+    local pid setup_script
+    pid="${1:-$(get_product_id)}"
+    for path in /usr/lib/{u-boot,edk2/*}/$pid/setup.sh; do
+        [[ -f "$path" ]] && setup_script="$path" && break
+    done
+
+    if [[ -z "${setup_script:-}" ]]; then
+        echo "This operation requires a setup script to work!" >&2
+        return 1
+    fi
+
+    echo "${setup_script:-}"
```

Comment:
```suggestion
    echo "${setup_script}"
```
到了这个分支就已经确定setup_script有值了